tests/test_mosaic.py

```diff
@@ -16,16 +16,17 @@
 from rio_tiler.models import ImageData, PointData
 from rio_tiler.mosaic.methods import defaults
 
-asset1 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_cog1.tif")
-asset2 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_cog2.tif")
-asset3 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_cog3.tif")
+asset1 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_value_1.tif")
+asset2 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_value_2.tif")
+asset3 = os.path.join(os.path.dirname(__file__), "fixtures", "mosaic_value_3.tif")
```

Comment:
I've updated the assets to avoid resampling to have a role (because depending on GDAL versions the pixel values can slightly change)